securedrop_client/queue.py

```diff
@@ -50,30 +53,23 @@ class RunnableQueue(QObject):
         ReplyDownloadJob: 17,
     }
 
-    '''
-    Signal that is emitted when processing stops
-    '''
+    # Signal that is emitted when processing stops
     paused = pyqtSignal()
 
-    '''
-    Signal that is emitted to resume processing jobs
-    '''
+    # Signal that is emitted to resume processing jobs
     resume = pyqtSignal()
 
     def __init__(self, api_client: API, session_maker: scoped_session, size: int = 0) -> None:
-        """
-        A size of zero means there's no upper bound to the queue size.
-        """
         super().__init__()
         self.api_client = api_client
         self.session_maker = session_maker
+        # If the maxsize provided to PriorityQueue is 0, there will be no upper bound
         self.queue = PriorityQueue(maxsize=size)  # type: PriorityQueue[Tuple[int, ApiJob]]
```

Comment:
since we were passing `maxsize` only for the metadata queue (which had maxsize=1) we can also remove this `maxsize` functionality if we want

Reply:
(can be done in a followup or at a later date)

also handling the `Full` exception in the job adding methods is only relevant when the queue can be full, so that can also be removed